print/cups/Portfile

```diff
@@ -0,0 +1,68 @@
+revision            0
+categories          print
+license             Apache-2
+maintainers         nomaintainer
+description         CUPS is the current standards-based, open source printing system developed by \
+                    OpenPrinting for Unix-like operating systems and uses IPP Everywhere™ to support \
+                    printing to local and network printers.
+long_description    OpenPrinting CUPS is the most current version of CUPS, a standards-based, \
+                    open source printing system for Unix-like operating systems. CUPS supports \
+                    printing to AirPrint™ and IPP Everywhere™ printers, \
+                    Network and local (USB) printers with Printer Applications, and \
+                    Network and local (USB) printers with (legacy) PPD-based printer drivers. \
+                    CUPS provides the System V ("lp") and Berkeley ("lpr") command-line interfaces, \
+                    a configurable web interface, a C API, and common print filters, drivers, and backends \
+                    for printing. The cups-filters project provides additional filters and drivers.
+homepage            https://openprinting.github.io/cups/
+
+checksums           rmd160  58a24fe177cfb66157899b6a53a90972a6080e74 \
+                    sha256  f03ccb40b087d1e30940a40e0141dcbba263f39974c20eb9f2521066c9c6c908 \
+                    size    8128785
+
+depends_build       port:gmake \
+                    port:autoconf \
+                    port:pkgconfig
+depends_lib         port:zlib
+
+patch {
+    reinplace "s|/System/Library/LaunchDaemons||g" configure
+}
+
+configure.cflags-append     -I${prefix}/include
+configure.cppflags-append   -I${prefix}/include
+configure.cxxflags-append   -I${prefix}/include
+configure.ldflags-append    -L${prefix}/lib
+configure.env-append    DSOFLAGS=-L${prefix}/lib
+configure.args      --disable-libusb \
+                    --disable-pam \
+                    --without-dnssd \
+                    --without-tls \
+                    --with-bundledir="" \
+                    --with-icondir=${prefix}/share/print/cups/icons \
+                    --with-ondemand="" \
+                    --with-rcdir=${prefix}/share/print/cups/etc/rc.d \
+                    --with-menudir=${prefix}/share/cups/applications \
+                    --bindir=${prefix}/bin \
+                    --sbindir=${prefix}/sbin \
+                    --libexecdir=${prefix}/libexec \
+                    --datarootdir=${prefix}/share/print/cups \
+                    --sysconfdir=${prefix}/share/print/cups/etc \
+                    --localstatedir=${prefix}/share/cups/var \
+                    --includedir=${prefix}/include \
+                    --oldincludedir=${prefix}/include \
+                    --docdir=${prefix}/share/doc/cups
+
+build.type          gnu
+build.target        all
+build.args          
+
+destroot.target     install
+destroot.post_args  BUILDROOT=${destroot}
```

Comment:
MacPorts sets `destroot.destdir` to `DESTDIR=${destroot}` by default. I presume this line here this means this build system does not honor `DESTDIR`? If so, it is customary to override `destroot.destdir` with whatever this build system requires (i.e. `BUILDROOT=${destroot}`) rather than use another variable like `destroot.post_args`.